tests/nightly/JenkinsfileForBinaries

```diff
@@ -106,6 +106,22 @@ core_logic: {
           utils.docker_run('ubuntu_nightly_gpu', 'nightly_tutorial_test_ubuntu_python3_gpu', true, '1500m')
         }
       }
+    },
```

Comment:
This should not be in the JenkinsfileForBinaries.

Reply:
Done. Moved to `tests/nightly/Jenkinsfile`. Thank you for your input.